docs/metrics.md

```diff
@@ -9,8 +9,8 @@ Sometimes the Help text on `/metrics` endpoint just isn't enough to explain what
 Kubevirt's version information
 
 Labels:
-* `goversion` - GO version used to compile this version of KubeVirt 
-* `kubeversion` - Git commit refspec that created this version of KubeVirt 
+* `goversion` - GO version used to compile this version of KubeVirt
```

Comment:
I cannot stand trailing spaces, but seeing them here is a distraction

Reply:
IIRC this is caused by `make generate`, failing to execute this would fail a sanity test

Reply:
If so, I would run in a separate commit.